packages/pds/src/service-db/migrations/index.ts

```diff
@@ -0,0 +1,5 @@
+import * as init from './001-init'
+
+export default {
+  ['001']: init,
```

Comment:
Basically nothing, just a tiny tweak for consistency with the other migration files.
```suggestion
  '001': init,
```